src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -67,17 +75,22 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
     return this._expandMode || 'none';
   }
 
+  @Output()
+  public activeIndexChange = new EventEmitter();
+
   @ContentChildren(SkyRepeaterItemComponent)
   public items: QueryList<SkyRepeaterItemComponent>;
 
   private ngUnsubscribe = new Subject<void>();
 
+  private _activeIndex: number;
```

Comment:
This can probably be a normal `Input` (no need for getter/setter).

Reply:
Reverted. Was leftover from before when I had it doing logic.